traitsui/qt4/list_editor.py

```diff
@@ -34,8 +34,7 @@
 from .editor import Editor
 from .helper import IconButton
 from .menu import MakeMenu
-from six.moves import range
-
+\
```

Comment:
Typo?

Reply:
Yep - UK keyboard with `\` where the return key rightfully should be.  Fortunately harmless in this instance.